docs/source/snakemake/debugging.md

```diff
@@ -0,0 +1,47 @@
+# Debugging Snakemake
+
+## Local Development
+
+When debugging a Snakemake workflow, it's helpful to run the JIT step locally instead of re-registering your workflow everytime you want to test a change. To address this, the Latch SDK supports local development for Snakemake workflows.
```

Comment:
After chatting with Fulcrum this was the #1 thing they requested. They said the development cycle is super slow for them because any bugs that they have in their Snakefile or input paths, causes them to have to re-register and then execute and wait for the JIT output to succeed/fail. This will allow them to verify that the JIT step passes without issues without having to do the register/execute/wait loop. 

For what its worth, I've been trying to get one of their snakemake workflows up and running this week and it was painful, definitely found it to be much faster after having the latch develop ability. 

It doesn't speed up the post JIT development time, but thats the next step.